### PR TITLE
Improve the rkt support in the Makefile

### PR DESCRIPTION
**Pull request by tomdee**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -129,8 +129,16 @@ dist/calicoctl:
 	chmod +x dist/calicoctl
 
 # Copy the plugin into place
-deploy-rkt: dist/calico
+deploy-rkt: binary
```
Comment (caseydavenport) [inline Makefile]: Why the switch from depending on the file, to the phony target?

Comment (tomdee) [inline Makefile]: So it builds the IPAM plugin too

**Comment by tomdee** [comment]
@caseydavenport This is still with you

**Comment by caseydavenport** [comment]
LGTM.

**Pull request merged**